docs/superpowers/specs/2026-09-04-profile-section-routes-design.md

```
@@ -0,0 +1,126 @@
+
+- No visual redesign. The tab strip keeps its current underline + icon styling.
+- No change to any of the 8 section components themselves (`ProfileForm`,
+  `CandidateContactsEditor`, `LocationCard`, `SkillsCard`, `ExperienceBankView`,
+  `EducationCard`, `ScreeningAnswersForm`, `AccountPreferences`).
+- No change to sibling `/my/**` sections (Tracking, Activity, Inbox, etc).
+
+## Design
+
+### Route layout
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)